plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/BasicAuthenticationDetailView.java

```diff
@@ -101,6 +102,9 @@ public boolean isViewFor(Object object) {
 	public void onVisible(IObservableValue detailsViewModel, DataBindingContext dbc) {
 		bindWidgetsToInternalModel(dbc);
 		chkRememberToken.setText("&Save password (could trigger secure storage login)");
+		setCheckboxVisible(true);
+		chkRememberToken.getParent().update();
+		chkRememberToken.getParent().layout(true);
```

Comment:
There are two checkboxes, one should be shown the other hidden. When they are toggled this way, parent is not obliged to update layout. I think it is not GTK3, just parent need to be explicitly notified that it has to change layout.